ionic-range-slider.js

```diff
@@ -7,7 +7,7 @@ angular.module("ion.rangeslider", []);
 angular.module("ion.rangeslider").directive("ionRangeSlider",
     function($timeout) {
         return {
-            restrict: "E",
+            restrict: "A",
```

Comment:
Yep, you have reason. Then, What is the better version for you about this change? 1.1 is better for you?
I you agree, I gonna change that.